guides/source/active_record_postgresql.md

```diff
@@ -110,6 +110,15 @@ profile.settings = {"color" => "yellow", "resolution" => "1280x1024"}
 profile.save!
 ```
 
+#### Querying HStore columns
```

Comment:
I don't think we need a separate heading. Simply expand the example given above. Also make sure to add a link to available functions and operators at the beginning of the `hstore` section.